.travis.yml

```diff
@@ -1,5 +1,10 @@
 language: csharp
-sudo: false
+sudo: required
+dist: trusty
```

Comment:
Doesn't this hit the issue in Mono that @BrennanConroy fixed? If yes, you probably need to wait for a KoreBuild workaround I'll be adding over the weekend, enabling `dnu restore` using DNX Core.